libraries/chain/controller.cpp

```diff
@@ -600,6 +601,25 @@ struct controller_impl {
          {"delay", cyberway::chaindb::tag<by_delay>::get_code(), true, {{"delay_until", "asc"}, {"id", "asc"}}},
          {"senderid", cyberway::chaindb::tag<by_sender_id>::get_code(), true, {{"sender", "asc"}, {"sender_id", "asc"}}}}
       });
+
+      // domain names
+      abi.structs.emplace_back( eosio::chain::struct_def{
+        "domain", "",
+        {{"id", "uint64"},
+         {"owner", "name"},
+         {"creation_date", "block_timestamp_type"},
+         {"name", "string"}}
+      });
+
+      abi.tables.emplace_back( eosio::chain::table_def {
+        "domain",
+        cyberway::chaindb::tag<domain_object>::get_code(),
+        "domain",
+        {{"id", cyberway::chaindb::tag<by_id>::get_code(), true, {{"id", "asc"}}},
+         {"string", cyberway::chaindb::tag<by_name>::get_code(), true, {{"name", "asc"}}},
+         {"name", cyberway::chaindb::tag<by_owner>::get_code(), true, {{"owner", "asc"}}}}
```

Comment:
Confused the type of field used as the name of this index. Maybe they should be called by the name of the index field?

Reply:
you're right, fixed locally, will add with following PR